lib/src/runner/configuration/load.dart

```diff
@@ -83,7 +81,41 @@ class _ConfigurationLoader {
   Configuration load() => _loadGlobalTestConfig()
       .merge(_loadLocalTestConfig())
       .merge(_loadGlobalRunnerConfig())
-      .merge(_loadLocalRunnerConfig());
+      .merge(_loadLocalRunnerConfig())
+      .merge(_loadIncludeConfig());
+
+  /// If an `include` node is contained in [node], merges and returns [config].
+  Configuration _loadIncludeConfig() {
+    var source = p.fromUri(_document.span.sourceUrl);
+    var includePathNode = _document.nodes["include"];
+    if (includePathNode != null && includePathNode.value is! String) {
+      throw new SourceSpanFormatException(
+          "The 'include' field must be omitted or a String",
+          includePathNode.span);
+    }
+    if (includePathNode != null) {
+      Uri includeValue;
+      try {
+        includeValue = Uri.parse(includePathNode.value);
+      } on FormatException catch (_) {
+        throw new SourceSpanFormatException(
+            "Not a valid file URL: \"${includePathNode.value}\"",
+            includePathNode.span,
+            source);
+      }
+      var includePath = p.fromUri(includeValue);
```

Comment:
You could write everything up to here as:

```dart
var includeNode = _document.nodes["include"];
if (includeNode == null) return Configuration.empty;

var includePath = _parseNode(includeNode, "include path", p.fromUri);
```

Reply:
Looks great, done!